src/components/AddTodoForm.jsx

```diff
@@ -10,8 +10,13 @@ const AddTodoForm = ({ onAddTodo }) => {
 
   const handleAddTodo = (e) => {
     e.preventDefault();
-    onAddTodo({ id: Date.now(), title: todoTitle });
-    setTodoTitle("");
+    const title = todoTitle.trim();
+    if (title !== "") {
```

Comment:
The statement `if (title !== "")` is equivalent to `if (title)`  in functionality because both check for a non-empty string.